be/src/olap/rowset/segment_v2/column_reader.cpp

```diff
@@ -1107,9 +1107,10 @@ Status FileColumnIterator::_read_data_page(const OrdinalPageIndexIterator& iter)
                 Slice dict_data;
                 PageFooterPB dict_footer;
                 _opts.type = INDEX_PAGE;
```

Comment:
If index is used, is there such case that we read page number 10 first and then read page number 1? Or in other words, could we make sure that the data page is read in descending order?

Reply:
> If index is used, is there such case that we read page number 10 first and then read page number 1? Or in other words, could we make sure that the data page is read in descending order?

no sure, I will figure it out somehow

Reply:
> If index is used, is there such case that we read page number 10 first and then read page number 1? Or in other words, could we make sure that the data page is read in descending order?

@yiguolei Yes, we are sure the data page is read in ascending order: after locating the first target page, we will read the following pages sequentially, for a single query of course.